content/docs/README.md

```diff
@@ -419,12 +419,18 @@ Example file structure:
 │   ├── architecture-overview.md
 ```
 
-Example content in `architecture-overview.md`:
+To display images using Markdown syntax, use the following syntax: `![alt text](image url)`. Example content in `architecture-overview.md`:
 
 ```md
 ![Neon architecture diagram](/docs/conceptual-guides/neon_architecture_2.png)
 ```
 
+To display image without border, use `"no-border"` as below:
```

Comment:
If you need an image without border to show an annotated piece of UI, use the `"no-border"` attribute as in the example below: